### PR TITLE
Add new source Affilbox and organization marketing.bi

### PR DESCRIPTION
**Pull request by banhetom**
I want to add new organization Marketing.bi and new source Affilbox
Could you please merge the code?

### CODE REVIEW

**Comment by banhetom** [comment]
@googlebot I signed it!

**Comment by banhetom** [comment]
@googlebot I fixed it.

File: sources/affilbox.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "id": "AFFILBOX",
+  "name": "AffilBox",
+  "categories": ["ADVERTISING"],
+  "organization": "MARKETING_BI",
+  "iconUrl": "https://www.affilbox.cz/wp-content/themes/affilbox2020/assets/images/favicon/apple-touch-icon.png",
+  "sourceUrl": "https://www.affilbox.cz/"
+}
```
Comment (pfrisella) [inline sources/affilbox.json:8]: `dataVisibility` property is missing.

See https://github.com/googledatastudio/ds-data-registry#add-a-source

**Review by pfrisella — CHANGES_REQUESTED** [review]

**Review by pfrisella — APPROVED** [review]
Thanks for updating. LGTM!

**Pull request merged**